### PR TITLE
REF: Language settings to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kebda3hl57rkag2inzgmsng6ya?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11685)

File: screen/settings/language.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, { useState, useEffect, useContext } from 'react';
```
Comment (Overtorment) [inline screen/settings/language.tsx:1]: View is not imported and thats why a bunch of tests failed..?

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nr3ps5mr7mmem6in4nkvkuwh24?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11697)

**Review by Overtorment — APPROVED** [review]
noice!

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**